devoir/pavage.ipynb

```diff
@@ -0,0 +1,313 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        contraintes=[]#Initialisation du talbeau de contraintes
</pre><p>Plus simple : <code>contraintes = [[]*n for _ in range(p)]</code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fortierq/oc-m1-2021/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>